helper.php

```diff
@@ -413,5 +415,21 @@ public function renameTag($formerTagName, $newTagName) {
         msg($this->getLang("admin renamed"), 1);
         return;
     }
-
+    
+    /**
+     * Deletes a tag
+     *
+     * @param array $tags
+     */
+    public function deleteTags($tags) {
+        if (empty($tags)) return;
+        
+        $db = $this->getDb();
+        
+        $query = 'DELETE FROM taggings WHERE ' . implode(' OR ', array_fill(0, count($tags), 'CLEANTAG(tag) = ?'));
```

Comment:
as discussed yesterday this should be adjusted to only apply to the current namespace filter